src/components/Input/index.tsx

```diff
@@ -39,11 +49,20 @@ export const Input = ({
           <input
             id={name}
             name={name}
-            type={type}
+            type={inputType}
             {...props}
             className={cn(inputStyle({ type, error }))}
             required={required}
           />
+          {type === "password" && (
+            <button
+              type="button"
+              onClick={togglePasswordVisibility}
+              className="absolute right-3 top-1/2 transform -translate-y-1/2 text-light-silverSteel hover:text-light-secondary dark:text-silverSteel dark:hover:text-secondary"
+            >
+              {showPassword ? <FaEyeSlash /> : <FaEye />}
+            </button>
+          )}
```

Comment:
@Honey0908 To me personally, This seems like a "special" scenario. Rather than bloating the component library with consumer specific logic, should we think in direction where we support icons on right or left side of input?

Reply:
And then, consumer can decide what they want to show based on the requirements?

Reply:
Example: https://www.chakra-ui.com/docs/components/input#left-and-right-element